sale_order_line_price_history/wizards/sale_order_line_price_history.py

```diff
@@ -106,7 +106,12 @@ class SaleOrderLinePriceHistoryline(models.TransientModel):
         related="sale_order_line_id.discount",
     )
 
+    def _prepare_sale_order_line_vals(self):
```

Comment:
This method name is too generic and can collapse with others. Call it `_prepare_set_price_history_vals`, and add a proper docstring. Don't lose good habits.